.travis.yml

```diff
@@ -42,6 +43,11 @@ script:
   - if [ $POD_LINT == "YES" ]; then
       pod lib lint;
     fi
+    
+  # Run `swift build` if specified
+  - if [ $SWIFT_BUILD == "YES"]; then
```

Comment:
I think we need to update the `matrix` line above to include `SWIFT_BUILD="YES"`, right?

Reply:
We technically don't even need these lines anymore. Swift build will fail since this library requires UIKit. I left them in just in case we wanted it in the future.

Reply:
Removed swift build section.